src/Pages/Form/Components/PatientInformationSection.tsx

```diff
@@ -0,0 +1,265 @@
+import "date-fns";
+import React, { useState } from "react";
+import {
+  createStyles,
+  makeStyles,
+  withStyles,
+  Theme,
+  ThemeProvider,
+  createTheme,
+} from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import MaleIcon from "../../../Assets/male.png";
+import FemaleIcon from "../../../Assets/female.png";
+import DateFnsUtils from "@date-io/date-fns";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#FFF",
+      contrastText: "#3A3A3D",
+    },
+    secondary: {
+      main: "#EF5DA8",
+      contrastText: "#FFF",
+    },
+  },
+});
+
+const useStyle = makeStyles((theme) =>
+  createStyles({
+    root: {
+      width: "100%",
+      marginLeft: "32px",
+    },
+    picker: {
+      marginTop: "34px",
```

Comment:
Please change all marginTop to marginBottom because in the project should be set either Top or Bottom. Previously, we use marginBottom.Therefore, this should follow.